### ISSUE TITLE
Feature request - prepend/append information to end of created files?

### ISSUE THREAD

**Issue by morrow95** [oklai/koala#754]
Would like to see the ability to prepend/append user input to the end of created files.  For instance, you modify a css file, it auto compiles and spits out the minified css file to where you specified, BUT you also are appending something like '-%unixtimestamp'  to the end of the file.  So a file called default.css would end up being compiled as default-1526918943.css.  Same goes for js files.  Would also remove the old file name and add the new.

Basically looking for a way to add versioning to the compiled css and js files.  I do not have the ability to use a pre-processor or any of the other 'typical' ways of handling this when publishing.
